### PR TITLE
Re-enable Preferred Language

### PR DESCRIPTION
**Pull request by jcarrigan2**

### CODE REVIEW

File: signup/signup.php [hunk 1]
```diff
@@ -81,12 +71,12 @@
 			<h3 class="form-subheading">Background Information</h3>
 
 			<ul class="dcf-pl-0">
-				<!-- TODO: NOTE THAT THIS WAS COMMENTED OUT, WE CURRENTLY DONT LET THEM SAY THE LANGUAGE THEY REQUIRED, WE LET IT DEFAULT TO ENGLISH
+				<!-- TODO: NOTE THAT THIS WAS COMMENTED OUT, WE CURRENTLY DONT LET THEM SAY THE LANGUAGE THEY REQUIRED, WE LET IT DEFAULT TO ENGLISH -->
```
Comment (MatthewMeacham) [inline signup/signup.php:74]: You could probably just remove this

File: signup/signupController.js [hunk 2]
```diff
@@ -89,8 +89,8 @@ define('signupController', [], function() {
 				};
 			}
 
-			// TODO: assume english for right now until support for other languages is added
-			$scope.data.language = "eng";
+			// If assuming english, comment out language div in signup.php and uncomment next line of code
+			// $scope.data.language = "eng";
```
Comment (MatthewMeacham) [inline signup/signupController.js:93]: You can probably just remove this

File: signup/signup.php [hunk 3]
```diff
@@ -81,12 +71,12 @@
...
 				<li class="form-radio" id="language">
 					<label for="language" class="form-required">Which language will you require?</label>
```
Comment (MatthewMeacham) [inline signup/signup.php:76]: Maybe workshop this text--could consider "Which language would you prefer?" or something. Maybe ask Tracy/Dr Moody on it

File: signup/signup.php [hunk 4]
```diff
@@ -81,12 +71,12 @@
...
 					<div>
 						<div class="dcf-btn-group" data-toggle="buttons">
-							NOTE: the values here are the ISO 639-2/T specfication for language codes (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)
+							<!-- NOTE: the values here are the ISO 639-2/T specfication for language codes (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) -->
 							<label class="dcf-btn dcf-btn-secondary"
```
Comment (MatthewMeacham) [inline signup/signup.php:80]: Are these 4 languages still the 4 they want displayed? Also, does this selection default to English?

**Review by MatthewMeacham — APPROVED** [review]
LGTM

File: signup/signup.php (hunk 1 above)
Comment (jcarrigan2) [reply signup/signup.php]: Removed it!

File: signup/signupController.js (hunk 2 above)
Comment (jcarrigan2) [reply signup/signupController.js]: Removed it!

File: signup/signup.php (hunk 4 above)
Comment (jcarrigan2) [reply signup/signup.php]: It defaults to English now.

Comment (jcarrigan2) [reply signup/signup.php]: Tracy just got back to me and let me know these are good options for now.

File: signup/signup.php (hunk 3 above)
Comment (jcarrigan2) [reply signup/signup.php]: I switched the wording and Tracy was good with it. Good call!

**Pull request merged**